tests/test_DGEList.py

```diff
@@ -232,7 +230,7 @@ def test_tpm():
     ensg_gene = icd.pick_gene_id(icd.get_genes_from_symbol(first_gene))
     gene_len = icd.get_length_of_canonical_transcript(ensg_gene)
 
-    # TPM=countsPerBase * (1/sum[countsPerBase]) * 10^6
+    # TPM = countsPerBase / sum[countsPerBase]) * 10^6
     assert tpm_dge.counts[0][0] == ((first_pos / gene_len) / rate_sum) * 1e6
```

Comment:
For some reason I can't find, the value for this test appears to be the correct calculation but off by 1000. I suspect it may be something related to a kilobase conversion somewhere, but I can't find it.